src/components/Square.js

```diff
@@ -1,19 +1,17 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import './Square.css'
+import './Square.css';
 
 const Square = (props) => {
   // For Wave 1 enable this 
   //  Component to alert a parent 
   //  component when it's clicked on.
-
-  return <button
-    className="square"
-  >
-    {props.value}
-  </button>
-}
+    const onClick = () => {
+        props.onClickCallback(props.id);
+    }
+    return <button onClick={onClick} id={props.id} key={props.id} className='square'>{props.value}</button>;
```

Comment:
instead of making another function you could just do
```suggestion
    return <button  
    onClick={() => onClickCallback(id)}
    id={props.id} 
    key={props.id} 
    className='square'
    >
    {props.value}
    </button>;
```